### DISCUSSION TITLE
How the Drill Down Work in Apache Superset

### DISCUSSION THREAD

**Discussion by Mudassarhu** [apache/superset#23510] [Q&A / Help] [answered]
Hello,
I have been creating the dashboard in which I have to use the drill down. When I click on any chart in Edit mode it does not show me the drill option. 
Is anything need to enable before drill down ? 
is I need to do something with preset ? 

What should I do ? 

Thanks

**Comment by mdeshmu** [comment, marked as answer]
Drill to detail is available in recent versions. 

Drill By is in development. 

You can read more here: https://preset.io/blog/drill-down-and-drill-by/
